agent/util-scripts/pbench-collect-sysinfo

```diff
@@ -104,9 +113,11 @@ function gather_remote_sysinfo_data {
 	local remote_host=$1
 	local remote_label=$2
 	debug_log "[$script_name]running sysinfo-dump on $remote_host"
-	cmd="ssh $ssh_opts -n $remote_host pbench-remote-sysinfo-dump $sysinfo_path $remote_label"
+	cmd="ssh $ssh_opts -n $remote_host pbench-remote-sysinfo-dump $sysinfo_path "$collect_sysinfo" "$tool_group_dir" $remote_label"
 	debug_log "[$script_name] $cmd > $sysinfo_path/$remote_label$remote_host.tar.xz 2> $sysinfo_path/$remote_label$remote_host.err"
-	$cmd > $sysinfo_path/$remote_label$remote_host.tar.xz 2> $sysinfo_path/$remote_label$remote_host.err
+	if [[ -n "$collect_sysinfo" ]] && [[ "$collect_sysinfo" != "none" ]]; then	
```

Comment:
Are we sure we want to test for the option here?  Why not test for it right after we gather the command line arguments, since if it is `none` there is no point in doing anything, right?  Perhaps around line 70?